src/schema/rules/checks/fmap.yaml

```diff
@@ -49,3 +49,16 @@ MagnitudeFileWithTooManyDimensions:
     - nifti_header != null
   checks:
     - nifti_header.dim[0] == 3
+
+EPIWithBValsHasHighBVals:
+  issue:
+    code: EPI_WITH_BVALS_HAS_HIGH_BVALS
+    message: |
+      EPI field maps with b-values should not include high b-values (in essence, b-values > 100).
```

Comment:
```suggestion
      EPI field maps with b-values should not include high b-values (defined as b-values > 100).
```

Reply:
Just a suggestion here. Also "high b-values" is sometimes understood to imply b > 1,000, where DTI fails. I wonder whether there is another phrase we could use here (sorry, can't come up with something good right now and short on time atm).

Reply:
I agree. @mattcieslak do you have ideas for wording?

Reply:
What about "should include only small b-values (defined as b-values <= 100)"?